content/docs/user-guide/managing-external-data.md

```diff
@@ -29,22 +29,22 @@ supported:
 > `dvc remote`.
 
 In order to specify an external output for a stage file, use the usual `-o` or
-`-O` options of the `dvc run` command, but with the external path or URL to the
-file in question. For <abbr>cached</abbr> external outputs (`-o`) you will need
-to [setup an external cache](/doc/command-reference/config#cache) in the same
-remote location. Non-cached external outputs (`-O`) do not require an external
-cache to be setup.
+`-O` options of `dvc run`, but with the external path or URL to the file in
+question. For <abbr>cached</abbr> external outputs (`-o`) you will need to
+[setup an external cache](/doc/use-cases/shared-development-server#configure-the-external-shared-cache)
+in the same external/remote file system first.
 
-> Avoid using the same remote location that you are using for `dvc push`,
-> `dvc pull`, `dvc fetch` as external cache for your external outputs, because
-> it may cause possible file hash overlaps: The hash value of a data file in
-> external storage could collide with that generated locally for another file.
+> Avoid using the same location of the
+> [remote storage](/doc/command-reference/remote) that you have for `dvc push`
+> and `dvc pull` for external outputs or as external cache, because it may cause
+> file hash overlaps: The hash value of a data file in external storage could
+> collide with the one generated locally for another file.
```

Comment:
Improved this note but it's still pretty strange. Difficult to understand what it is about. The ambiguity of the term "remote" is giving us problems here. Not sure how to address, maybe just remove the note?

Reply:
yep, it's a very complicated to simplify it :) the whole topic is very advanced and very technical. Let's keep it for now with your revisions and get back to it when we have more time to spend on all this data management problems.